src/crypto/hsm/nxp/PersistentStorageOperationalKeystoreHSM.cpp

```diff
@@ -0,0 +1,279 @@
+/*
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "PersistentStorageOperationalKeystoreHSM.h"
+#include "CHIPCryptoPALHsm_SE05X_utils.h"
+#include <crypto/OperationalKeystore.h>
+#include <crypto/hsm/CHIPCryptoPALHsm.h>
+#include <lib/core/CHIPError.h>
+#include <lib/core/CHIPPersistentStorageDelegate.h>
+#include <lib/core/CHIPTLV.h>
```

Comment:
Does this file use persistent storage or TLV?  I don't think it does.

Reply:
Yes. Neither of them are used. Removed.